src/server/pkg/storage/chunk/util.go

```diff
@@ -2,16 +2,22 @@ package chunk
 
 import (
 	"math/rand"
+	"testing"
 
+	"github.com/jmoiron/sqlx"
 	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
+	"github.com/pachyderm/pachyderm/src/server/pkg/storage/tracker"
 )
 
-// WithLocalStorage creates a local storage instance for testing during the lifetime of
+// NewTestStorage creates a local storage instance for testing during the lifetime of
 // the callback.
-func WithLocalStorage(f func(obj.Client, *Storage) error, opts ...StorageOption) error {
-	return obj.WithLocalClient(func(objClient obj.Client) error {
-		return f(objClient, NewStorage(objClient, opts...))
-	})
+func NewTestStorage(t testing.TB, db *sqlx.DB, tr tracker.Tracker, opts ...StorageOption) (obj.Client, *Storage) {
+	// metadata store
+	db.MustExec(schema)
```

Comment:
This should probably be replaced with a function similar to `fileset.NewTestStore`.

Reply:
We need a way to inject the tracker, since in order for filesets to depend on chunks they have to share a tracker. It also makes sense to check that everything works on a single DB.

Reply:
See the function `newTestStorage` in `chunk_test.go`, which only takes `testing.TB` and is used for the chunk package tests, but would not be useful for export.

Reply:
I was just referring to the schema setup and store constructor. For file sets, we have one function (`NewTestStore`) that does both, which takes a db and returns a store.